AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -389,11 +389,10 @@ - (void)onDidLoad:(ASDisplayNodeDidLoadBlock)body
 {
   ASDN::MutexLocker l(__instanceLock__);
   if ([self _isNodeLoaded]) {
-    ASDisplayNodeFailAssert(@"Attempt to call %@ on node after it was loaded. Node: %@", NSStringFromSelector(_cmd), self);
-    return;
-  }
-  
-  if (_onDidLoadBlocks == nil) {
+    ASDisplayNodeAssertThreadAffinity(self);
+    ASDN::MutexUnlocker l(__instanceLock__);
+    body(self);
+  } else if (_onDidLoadBlocks == nil) {
```

Comment:
Changed this behavior to call `-didLoad` before any `onDidLoadBlocks`, and no longer to throw if they call on main on a loaded node (just runs block).